0004-metric-configurable-aggregation.md

```diff
@@ -0,0 +1,73 @@
+# Let Metrics support configrable, recommended aggregations
```

Comment:
We need to allow users to configure the aggregation for the "raw statistics" as well as the set of labels. In the OpenCensus case the full set of labels supported are Tags (available in the context when the record method is called) + extraLables (specifically recorded with every request), and users should define a subset from there.